integration-tests/internals.test.js

```diff
@@ -0,0 +1,70 @@
+const { queryByText, fireEvent, waitFor, getByLabelText } = require('@testing-library/dom');
+const { default: userEvent } = require('@testing-library/user-event');
+const { startAppAndWait } = require('./test-helpers');
+
+/**
+ * The following suite of tests verify the behavior of the internals of Tram-One, more so than other tests might.
+ * They are often inpercievable to end-users, and verify the expected behavior of the behind-the-scenes design.
+ */
+
+describe('Tram-One', () => {
+	it('should clean up stores for elements that are no longer rendered', async () => {
```

Comment:
this is the more nitty-gritty, internals-aware version a test that we have in the regressions suite. This was more of a stepping stone for those other tests, and while slightly redundant, it acts as a good sanity check if those other tests end up failing. 

In reality, if the internals of the app change, this could end up needing to be scrapped, but we'll cross that bridge when we get there.